src/lib/workspace/AnalysisPanel.tsx

```diff
@@ -87,6 +87,7 @@ interface Props {
   /** API client that will be used in the Download Tab */
   downloadClient: DownloadClient;
   singleAppMode?: string;
+  showUnreleasedData: boolean;
```

Comment:
Good that it's not an optional prop :+1: I don't have to check it's provided everywhere!